fiware/CMakeLists.txt

```diff
@@ -19,14 +19,26 @@ cmake_minimum_required(VERSION 3.5.0)
 project(soss-fiware)
 
 find_package(soss-core REQUIRED)
+find_package(yaml-cpp REQUIRED)
+include(FindCURL)
+find_package(curl REQUIRED)
```

Comment:
This three lines should have an if(WIN32). They output an error in LINUX

Reply:
Done